src/modules/iotjs_module_profiler.c

```diff
@@ -1,5 +1,14 @@
 #include "iotjs_def.h"
 
+JS_FUNCTION(TakeSnapshot) {
+  if (!jerry_value_is_string(jargv[0]))
+    return JS_CREATE_ERROR(COMMON, "filepath should be required.");
+
+  iotjs_string_t filepath = JS_GET_ARG(0, string);
+  jerry_heap_profiler_take_snapshot ((const char*)iotjs_string_data(&filepath));
```

Comment:
Remove the space after the function `jerry_heap_profiler_take_snapshot`, that might be only in the JerryScript source.